wecSimTest.m

```diff
@@ -1,3 +1,51 @@
+function results = wecSimTest(options)
+
+    arguments
+        options.bemioTest = true
+        options.regressionTest = true
+        options.passiveYawTest = false
+        options.compilationTest = true
+        options.runFromSimTest = true
+        options.rotationTest = true
+    end
+    
+    import matlab.unittest.TestSuite;
+    import matlab.unittest.Test;
+    import matlab.unittest.TestRunner;
+    import matlab.unittest.plugins.*;
```

Comment:
This is a general programming comment, but it's normally bad style to use `import *` due to name space pollution (i.e. importing everything in the parent module). Now MATLAB is one huge namespace pollution, so it makes little difference, but if you were going to be programming in Python it's considered bad because you can't trace back the source of the functions / classes you are calling to the module you imported from. It would be better to add the plugins explicitly, e.g.

```
import matlab.unittest.plugins.DiagnosticsRecordingPlugin
import matlab.unittest.plugins.CodeCoveragePlugin
```

Reply:
Fair enough, I'll change this. As you say, its better practice to avoid the ``import *`` even if it doesn't make much difference in MATLAB.